dataland-frontend/src/components/resources/frameworkDataSearch/p2p/P2pDataModel.ts

```diff
@@ -667,8 +667,8 @@ export const p2pDataModel = [
           {
             name: "buildingEnergyEfficiency",
             label: "Building energy efficiency",
-            description: "Average energy requirement in kWh/m²a",
-            unit: "",
+            description: "Average energy requirement",
+            unit: "kWh/m²a",
```

Comment:
Again, Unit now no longer visible in upload page. Only acceptable if this will be resolved as part of DALA-2373.